expr/helper/helper.go

```diff
@@ -147,11 +147,11 @@ func AggregateSeries(name string, args []*types.MetricData, absent_if_first_seri
 }
 
 // SummarizeValues summarizes values
-func SummarizeValues(f string, values []float64) (float64, bool) {
+func SummarizeValues(f string, values []float64) (error, float64, bool) {
```

Comment:
By convention, errors should be the last return value. See https://github.com/golang/lint/blob/738671d3881b9731cc63024d5d88cf28db875626/lint.go#L1245

Reply:
Done!